addons/database_shrinking.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2021, https://www.hanssonit.se/
+# Copyright © 2021 Simon Lindner (https://github.com/szaimen)
+
+true
+SCRIPT_NAME="Database Shrinking"
+# shellcheck source=lib.sh
+source /var/scripts/fetch_lib.sh || source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+SCRIPT_EXPLAINER="This script allows to shrink your database if it has grown too much due to the usage of external storage.
+If you don't use external storage, you should NOT run this script!"
+
+# Variables
+DAILY_BACKUP_FILE="$SCRIPTS/daily-borg-backup.sh"
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Check if root
+root_check
+
+# Show install popup
+install_popup "$SCRIPT_NAME"
+
+# Backup
+msg_box "It is recommended to make a backup and/or snapshot of your NcVM before shrinking the database."
+if ! yesno_box_no "Have you made a backup of your NcVM? If yes, we will start with the database shrinking now."
+then
+    if ! [ -f "$DAILY_BACKUP_FILE" ]
+    then
+        exit 1
```

Comment:
Maybe you should add that you need `DAILY_BACKUP_FILE="$SCRIPTS/daily-borg-backup.sh"` to continue instead of just exiting. 

Also, and instruction how to set it up.

Reply:
Should be fixed with https://github.com/nextcloud/vm/pull/2141/commits/9dd7536c5bcdc2ae0a0a8ef6f930fad0348a24ed